sdk/python/arvados/arvfile.py

```diff
@@ -516,7 +517,7 @@ def _block_prefetch_worker(self):
                     return
                 self._keep.get(b)
             except Exception:
-                pass
+                _logger.error(traceback.format_exc())
```

Comment:
Maybe include "Warning:" in the exception message as well so that people know this is not an expected exception.